package.json

```diff
@@ -7,8 +7,8 @@
   ],
   "packageManager": "pnpm@7.20.0",
   "engines": {
-    "node": ">=18.x",
-    "pnpm": ">=7.0.0"
+    "node": ">=v18.12.1",
```

Comment:
Why should we force for this specific version instead of `v18.x` ?

Reply:
See https://github.com/scaleway/scaleway-ui/pull/2161#discussion_r1060101653

Reply:
Update: https://github.com/scaleway/scaleway-ui/pull/2161#issuecomment-1369775010